packages/core-sdk/src/types/resources/royalty.ts

```diff
@@ -185,6 +74,46 @@ export type ClaimAllRevenueRequest = {
     autoUnwrapIpTokens?: boolean;
   };
 };
+export type BatchClaimAllRevenueRequest = WithClaimOptions & {
+  /** The ancestor IPs from which the revenue is being claimed. */
+  ancestorIps: {
+    /** The address of the ancestor IP from which the revenue is being claimed. */
+    ipId: Address;
+    /**
+     * The address of the claimer of the currency (revenue) tokens.
+     *
+     * This is normally the ipId of the ancestor IP if the IP has all royalty tokens.
+     * Otherwise, this would be the address that is holding the ancestor IP royalty tokens.
+     */
+    claimer: Address;
```

Comment:
can we try to reuse `ClaimAllRevenueRequest` properties so we don't have to duplicate the jsdoc?
So could be something like:
```
ancestorIps: Omit<ClaimAllRevenueRequest, 'ancestorIpId' | "wipOptions"> & { 
 /** The address of the ancestor IP from which the revenue is being claimed. */
 ipId: Address
}
```